### PR TITLE
Cat utils patch

### PR DESCRIPTION
**Pull request by SunilSimha**
A bunch of fixes and improvements for catalog slurping tools and photometry processing wrappers.

### CODE REVIEW

File: frb/surveys/des.py [hunk 1]
```diff
@@ -141,6 +141,7 @@ def _select_best_img(self,imgTable,verbose,timeout=120):
 
         """
         row = imgTable[np.argmax(imgTable['exptime'].data.data.astype('float'))] # pick image with longest exposure time
+        import pdb; pdb.set_trace()
```
Comment (profxj) [inline frb/surveys/des.py:144]: remove?

**Review by profxj — APPROVED** [review]
Thanks for all of this!

I made one suggestion.
Also, please make sure the unit tests are passing.

**Review by profxj — APPROVED** [review]
thx!

**Comment by profxj** [comment]
Heads up that there is a conflict to resolve before we can merge.

**Comment by SunilSimha** [comment]
There are still a few kinks in the code, and Michelle is helping me as an alpha tester. :)
I am hoping to clear the conflicts and add unit tests this weekend.

File: frb/surveys/des.py (hunk 1 above)
Comment (SunilSimha) [reply frb/surveys/des.py:144]: Done.

**Pull request merged**